src/calibre/srv/render_book.py

```diff
@@ -639,6 +639,12 @@ def work_priority(name):
     spineq = frozenset(spine)
     landmarks = [l for l in get_landmarks(container) if l['dest'] in spineq]
 
+    page_progression_direction = None
+    try:
+        page_progression_direction = container.opf_xpath('//opf:spine/@page-progression-direction')[0]
+    except IndexError:
+        pass
```

Comment:
Hope I did it right. This was based on the guidance from your email. Thanks again!